packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,97 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "PIR Control",
+			"maxNodes": 4
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Report Value Sent to Associated Devices",
+			"description": "Set the value of basic set report sent to the light associated with the sensor in Group 2 (so the light turns on to the selected brightness level). 99 is full brightness.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Enable / Disable Motion Reports",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled (default)",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "Motion Sensitivity",
+			"description": "Adjust motion sensitivity where 10 is the most sensitive setting.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Lux Level Trigger",
+			"description": "Set lux level trigger with 10 being the lowest reported value.  The sensor will report motion only if lux level is below the set value. 0=set manually; 1=ignore lux.",
```

Comment:
> . 0=set manually; 1=ignore lux.

I think these should be options

Reply:
I don't think this is do-able, since allowManualEntry has to be true.  User can select 0 (set manually), 1 (ignore lux), or 10-900 for the trigger (threshold) level.  (Actually unclear what happens if user selects 2-9.)

Reply:
Yeah, it is. Options may exist always, but if `allowManualEntry` is `true`, they **must** exist.

Reply:
OK, I added the options and included some additional ones to be in concordance with the read-only parameters 7 and 8 (i.e., to match what can physically be set by the knobs on the sensor).

Reply:
@AlCalzone and @blhoward2 So I had a chance to mess with this some, and here is what I discovered (on version 1.3.0). If I include options and allowManualEntry is true, the options don't show up in the interface:  as far as I could tell, I could *only* enter the value manually, and there was no dropdown.  

Also (with regard to parameters 7 and 8), if I have options set for a read-only parameter (and allowManualEntry is set to true) - if the actual value isn't one of the options, the value doesn't show up.